html/semantics/forms/form-submission-0/form-submission-algorithm.html

```diff
@@ -18,16 +18,33 @@
   assert_equals(counter, 2);
 }, 'If constructing entry list flag of form is true, then return');
 
-let test10 = async_test('Cannot navigate (after constructing the entry list)');
-test10.step(() => {
+
+async_test(t => {
```

Comment:
Do you think this style of test is better than the counter style used above? If so, maybe add a comment?

Reply:
I'd like to test which one of two ```click()```s submits the form.  I'll add a comment.